src/libs/ReportActionsUtils.js

```diff
@@ -41,14 +41,16 @@ function isDeletedAction(reportAction) {
 }
 
 /**
- * Returns the object of reportActions
+ * Returns the parentReportAction if the given report is a thread.
  *
- * @param {String} reportID
+ * @param {Object} report
  * @returns {Object}
  */
-function getReportActions(reportID) {
-    const reportAction = lodashGet(allReportActions, [reportID]);
-    return reportAction;
+function getParentReportAction(report) {
+    if (!report || !report.parentReportID || !report.parentReportActionID) {
+        return {};
+    }
```

Comment:
This can be replaced by `ReportUtils.isThread`.

Reply:
That causes a cyclic import if we import `ReportUtils`

Reply:
Oh right good point.